argo/application-field-validation/artifacthub-pkg.yml

```diff
@@ -0,0 +1,21 @@
+name: application-field-validation
+version: 1.0.0
+displayName: application field validation
```

Comment:
This should take the value verbatim from the policy annotation `policies.kyverno.io/title`.